hijack/helpers.py

```diff
@@ -128,15 +139,24 @@ def login_user(request, hijacked):
     backend = get_used_backend(request)
     hijacked.backend = "%s.%s" % (backend.__module__, backend.__class__.__name__)
 
+    pre_hijack_started_results = pre_hijack_started.send(
+        sender=None, request=request,
+        hijacker=hijacker, hijacked=hijacked,
+        # send IDs for backward compatibility
+        hijacker_id=hijacker.pk, hijacked_id=hijacked.pk
+    )
+
     with no_update_last_login():
         # Actually log user in
         login(request, hijacked)
 
     hijack_started.send(
-            sender=None, request=request,
-            hijacker=hijacker, hijacked=hijacked,
-            # send IDs for backward compatibility
-            hijacker_id=hijacker.pk, hijacked_id=hijacked.pk)
+        sender=None, request=request,
+        hijacker=hijacker, hijacked=hijacked,
+        # send IDs for backward compatibility
+        hijacker_id=hijacker.pk, hijacked_id=hijacked.pk,
```

Comment:
Since you are adding a new signal, I don't see the need to have backwards compatible behavior.